concent_api/core/tests/test_payments_database_transaction_storage.py

```diff
@@ -52,7 +59,7 @@ def _create_transaction(self):
         )
 
     def test_that_initial_nonce_should_be_retreived_from_rpc(self):
```

Comment:
"retrived from rpc"? what is "rpc"? ;)

Reply:
It means that we expect to get (from this example nonce=5) from rpc which instance we have in singleton